.github/workflows/ci.yaml

```diff
@@ -232,13 +225,16 @@ jobs:
         name: Push builder images to ECR
         run: |-
           echo Pushing image $IMAGE_NAME-builder to ECR.
-          docker push $IMAGE_NAME-builder
+          docker buildx build --push --target builder \
+            --cache-to type=registry,ref=$IMAGE_NAME-builder,mode=max,image-manifest=true \
```

Comment:
- `image-manifest` needed to push buildcache to ECR (see [here](https://github.com/aws/containers-roadmap/issues/876#issuecomment-1665121877))
- `mode=max` is just a safety to make sure all the layers are exported to the buildcache (not tested if necessary but have needed in other projects)